nima_core/capture_message.py

```
@@ -0,0 +1,101 @@
+        
+        # Output result
+        output = {
+            "success": result,
+            "who": args.who,
+            "what": what,
+            "importance": args.importance,
+            "type": args.memory_type
+        }
+        print(json.dumps(output))
+        
+        return 0 if result else 1
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)